x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/package_policy_input_var_field.tsx

```diff
@@ -178,6 +187,7 @@ export const PackagePolicyInputVarField: React.FunctionComponent<{
       isEditPage,
       isInvalid,
       fieldLabel,
+      options,
```

Comment:
I was suspicious about memoizing an array since it would be compared by reference, but I'm not sure at all if that could be an issue in this case. I didn't see any strangeness with local testing. Does anyone know?